job/job.go

```diff
@@ -166,9 +172,39 @@ func (j *Job) UnmarshalJSON(b []byte) error {
 		} else {
 			err = errors.New("MimeType pattern must be a string")
 		}
-		return err
+		if err != nil {
+			return err
+		}
+	} else {
+		j.MimeType = ""
```

Comment:
This clause is not needed - j.MimeType is already set to "".

For reference: https://golang.org/ref/spec#The_zero_value.